service/src/main/java/com/epam/esm/services/constants/CertificateSearchParameters.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.esm.services.constants;
+
+public enum CertificateSearchParameters {
+    NAME, TAGNAME, DESCRIPTION, PRICE_LESS, PRICE_MORE;
+
+    public static CertificateSearchParameters getEntryByParameter(String s) {
+        return valueOf(s.toUpperCase().replace("-", "_"));
```

Comment:
string conversion in method "get".   String can be formatted before get method. And get method will return just value, without additional actions.

Reply:
this method only adds String conversion to valueOf() method, Without conversion it vill be just valueOf. Although can move string conversion to EntityService or Services, but it will cause appearance of such constructions as CertificateSearchParameters.valueOf(convertParameter(parameter))
instead of 
CertificateSearchParameters.getEntryByParameter(parameter)
May I just rename method?